modules/top-level/nixpkgs.nix

```diff
@@ -0,0 +1,78 @@
+{
+  config,
+  options,
+  lib,
+  ...
+}:
+let
+  cfg = config.nixpkgs;
+  opt = options.nixpkgs;
+in
+{
+  options.nixpkgs = {
+    pkgs = lib.mkOption {
+      # TODO:
+      # defaultText = lib.literalExpression ''
+      #   import "''${nixos}/.." {
+      #     inherit (cfg) config overlays localSystem crossSystem;
+      #   }
+      # '';
```

Comment:
It probably makes sense to include a defaultText explaining the current behaviour:

```nix
defaultText = literalMD ''
  The `pkgs` inherited from your host config (i.e. NixOS, home-manager, or nix-darwin),
  or the `pkgs` supplied to `makeNixvimWithModule` when building a standalone nixvim.

  > [!CAUTION]
  > This default will be removed in a future version of nixvim
''
```

Reply:
I've added this, it now renders the default as:
![image](https://github.com/user-attachments/assets/fac896d5-e71b-4cfb-aeef-51127cf976ca)